CONTRIBUTING.md

```diff
@@ -92,6 +93,72 @@ Your PR can contain more than one patch. Use as many patches as necessary to
 different items in your patches or PR, they will likely need to be reworked.
 
+### Assisted PR workflow
+
+If your PR is deemed useful but you are struggling to update it based on
+review feedback, or your PR appears to have been abandoned, the project
+maintainers may suggest (or decide in your absence) that an assisted PR
+workflow be employed. This involves creating a new PR using your PR as a base
+and applying fixes to it.
+
+The full workflow:
+
+1. A team member will be assigned to handling the assisted PR.
+
+1. The team member will add a comment on the original PR explaining what is
+   about to happen:
+
+   > Thank you for your contribution but we are unable to merge it until all
+   > review feedback has been applied and all CI checks have passed.
+   >
+   > Since we have not heard from you for a while, we assume you are unable to
+   > progress this PR. As such, we intend to create a new PR based on this
+   > one, and resolve the identified issues on the new PR. The new PR will
+   > reference this PR for tracking purposes.
+   >
+   > This PR will be marked `do-not-merge` and will be closed when our new PR
+   > has been merged. We will credit all authors of this PR on the new PR.
+
+1. The team member will apply the `do-not-merge` label to the original PR.
+
+1. The team member will create a new PR based on the contents of the original
+   PR.
+
+   The commits from the original PR should be retained where possible.
+
+   At least one commit in the new PR should reference the old PR or its
+   associated issue.
+
+   All authors of the new PR (which includes all authors of the original PR)
+   should be credited using a `Signed-off-by` comment for each author.
+
+1. The team will update the new PR to resolve the issues with the original PR.
+
+   They may decide to do this work collaboratively if it makes sense to do so.
+   In this scenario multiple people will push commits to the same PR branch.
+
+   This approach may be adopted if the number of changes required is high, or
+   the changes can be split into logical and discrete groups, or the PR requires
+   input from multiple team members, or the PR is urgent and needs to be resolved
+   as soon as possible. This last case can occur, for example, with
+   [documentation PRs](Documentation-Review-Process.md) where it is essential
+   the documenation be correct.
+
+1. Once the creator of the new PR is happy the new PR resolves all the issues
+   with the original PR, they will request the new PR is reviewed by the team
+   *and the original PR creator*.
+
+   Once all the standard approvals are obtained, the new PR will be merged.
+
+   The original PR creators approval is recommended but optional; if they do
+   not respond in a timely fashion (generally a week, but maybe a much shorter
+   period for urgent PRs), the PR will be merged regardless.
+
+   If the original PR creator subsequently disagrees with the contents of the
```

Comment:
Lines 157-158 suggested rewrite: 
If the original PR creator disagrees with the contents of the new PR after it is merged, they can raise a PR to resolve the issues.